src/components/composite/entities/Communication/DiscussionIcon.tsx

```diff
@@ -0,0 +1,30 @@
+import { SvgIconProps } from '@material-ui/core';
+import { AllInclusive, EmojiObjects, Forum, HelpOutline, Share } from '@material-ui/icons';
+import { createElement, FC, useMemo } from 'react';
+import { DiscussionCategoryExt, DiscussionCategoryExtEnum } from '../../../../models/enums/DiscussionCategoriesExt';
+import { DiscussionCategory } from '../../../../models/graphql-schema';
+
+interface DiscussionIconProps extends SvgIconProps {
+  category: DiscussionCategoryExt;
+}
+
+export const DiscussionIcon: FC<DiscussionIconProps> = ({ category, ...rest }) => {
```

Comment:
we have to sync the method for the icons. also I believe the design uses the outlined version of the icons